packages/frontend/core/src/components/affine/setting-modal/general-setting/experimental-features/config.ts

```diff
@@ -0,0 +1,72 @@
+import type { AppSetting } from '@toeverything/infra';
+
+export type BuildChannel = 'stable' | 'beta' | 'canary' | 'internal';
+
+export type FeedbackType = 'discord' | 'email' | 'github';
+
+export type Flag<K extends string> = Partial<{
+  [key in K]: {
+    displayName: string;
+    displayChannel: BuildChannel[];
+    description?: string;
+    feedbackType?: FeedbackType;
+    displayInClientOrWeb?: 'client' | 'web';
```

Comment:
How about change it to `restrictedPlatform`? The wording seems weird to me and confusing